### PR TITLE
Fixed Bug and Image Issue permisssion location

### PR DESCRIPTION
**Pull request by link2rn**
**Hello Sandip,** 
I've changed your code to make it more dynamic and efficient. Changes I've made are:

1. Upload Folder to the dynamic location where your project lies not only to your PC
2. Get Credentials from .aws folder. You can add your session on .aws folder itself
3. Changes Image Save location and fixed them in code
4. Added requirements.txt so that anyone can run your Project by adding the required library
5. AJAX issue to change image location dynamically. Before this, your code use to show the previous cache so I cleared the ace on each new image is stored.
6. - Other Minor

If you feel this would be a good contribution to your code, feel free to merge the pull request.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -69,7 +63,7 @@ def detect():
         
     IMG_TAIL = random.randint(0, 1000000000)
     file.save(os.path.join(UPLOAD_FOLDER, str(IMG_TAIL) + 'result.png'))
-    return jsonify(response['FaceDetails'], str(IMG_TAIL) + 'result.png')
+    return jsonify(response['FaceDetails'], 'image.png')
```
Comment (Sun-Deep) [inline app.py:66]: I think this `file.save(os.path.join(UPLOAD_FOLDER, str(IMG_TAIL) + 'result.png'))` should be replaced by `file.save(os.path.join(UPLOAD_FOLDER, 'image.png'))`. 
Isn't it ?

Comment (link2rn) [reply app.py:66]: Yes

Comment (Sun-Deep) [reply app.py:66]: Can you fix ? Then I will merge this pull request.

Comment (link2rn) [reply app.py:66]: You can merge them into your repo and update the changes. As I've already dumped repo from my local system.

Comment (Sun-Deep) [reply app.py:66]: Okay. Thank you for the contribution.

**Pull request merged**